### ISSUE TITLE
Add JPMS module descriptor

### ISSUE THREAD

**Issue by A248** [TheElectronWill/night-config#92]
NightConfig does not include a module descriptor for use with JPMS. As such, dependents using the Java module system cannot rely on the unstable module name.

At the absolute minimum, NightConfig should declare an `Automatic-Module-Name`. Ideally, it should include a full module descriptor.

**Comment by TheElectronWill** [comment]
Hasn't this been solved by #84 ?

**Comment by A248** [comment]
My bad, I only noticed #50 but saw that it was closed.

If you don't mind my asking, do you intend to make a release in the near future which declares a stable module name?

I also noticed that #84 was merged to the stable-3.x branch and not the master branch. Is that relevant?

**Comment by TheElectronWill** [comment]
Yes, I should do a release soon (in a few days/weeks), which should include #84 :)

The master branch is the unstable next version of NightConfig, I will publish from the `stable-3.x` branch.

**Comment by A248** [comment]
Thanks, I appreciate your work. I've double-checked and the module names are valid (no dashes).
